### ISSUE TITLE
UI doesn't show plugin list upon loading

### ISSUE THREAD

**Issue by davidmartinezanim** [pyblish/pyblish-qml#57]
I've noticed that when I open the UI, the list of plugins does not show up. When that happens, resizing the window forces the interface to re-draw and show the plugin names.

I think that the reason is the amount of plugins - This is the only thing that changed for me in the last few days

**Comment by mottosso** [comment]
I'm having this too, not sure of the cause, but it happens with a small amount of objects too.

In the console, when it happens, I'm getting this.

```python
QObject::connect: Cannot queue arguments of type 'QQmlChangeSet'
(Make sure 'QQmlChangeSet' is registered using qRegisterMetaType().)
```

Which might suggest that a connection is being made before something is fully loaded.

Researching..

**Comment by mottosso** [comment]
This was fixed in 0.2.5. 

It was related to populating a QAbstractItemModel from a thread. Let me know if you're still having this issue!
